runtime/etc/Makefile.mli-shared

```diff
@@ -69,13 +75,15 @@ $(TMPBINNAME): $(CHPL_CL_OBJS) FORCE
 		-L$(CHPL_LN_LIB_DIR) \
 		-lm -lchpllaunch \
 		$(LAUNCH_LIBS) \
-		$(CHPL_ZMQ_LIB_SEARCH_PATH) \
-		-lzmq
+		-lzmq \
```

Comment:
I realize this is just personal taste, because all `-L` (cap L) linker options on the entire linker command line apply to all `-l` (little L) options, but I still find it a lot easier to make sense of what's going on if the linker dir option that says where a library can be found comes before that library option on the command line. So I'd retain the original order here, with the `-lzmq` last. (Note: definitely not required for merge.)